### PR TITLE
Added Wowza Page

### PR DESCRIPTION
**Pull request by JakeDawkins**
*I figured I'd just open this as a PR to build on as we do the wowza stuff

# Feature / Fixed Issue(s)
- Added a blank page for wowza player testing
- Added route as `/wowza/:embedCode`

# Testing/Documentation
- [x] All significant new logic is covered by tests.
- [ ] Changelog has been updated.

# Screenshots
screen shot 2017-02-13 at 3 45 52 pm

### CODE REVIEW

**Comment by jbaxleyiii** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://080bad7e-0483-4898-8e71-f4c9db939d55.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/NewSpring/Holtzman/branch/wowza-page/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

File: imports/components/@primitives/UI/live/index.js [hunk 1]
```diff
@@ -79,14 +81,24 @@ class LiveWithoutData extends Component {
...
+    // create beta link
+    const shouldShowBetaLink = (
+      this.props.person
+      && !this.props.person.authLoading
+      && this.props.person.authorized
+    );
+    const link = shouldShowBetaLink ? `/wowza/${embedCode}` : `/video/${embedCode}`;
+
     return (
       <Motion
```
Comment (JakeDawkins) [inline imports/components/@primitives/UI/live/index.js]: @jbaxleyiii any thoughts on testing this? Right now, [the snapshots](https://github.com/NewSpring/Holtzman/blob/master/imports/components/%40primitives/UI/live/__tests__/__snapshots__/index.js.snap#L23) don't actually render out the link, so I can't think of a way to check the redirect for beta testers. I messed with this for a while today, and I can't seem to mock `Motion` and get the function that is has as a child to call and render `Link`

Comment (jbaxleyiii) [inline imports/components/@primitives/UI/live/index.js]: @JakeDawkins move the logic of find a link to a method on the class and use it inside the render function. Then you can test the method

File: imports/components/@primitives/UI/live/index.js [hunk 2]
```diff
@@ -113,11 +131,28 @@ const map = ({ live }) => ({ live });
 
 const withRedux = connect(map);
 
+// const withLive = graphql(LIVE_QUERY, {
+//   options: { pollInterval: 60000 },
+// });
+
+// XXX FOR TESTING ONLY
 const withLive = graphql(LIVE_QUERY, {
```
Comment (jbaxleyiii) [inline imports/components/@primitives/UI/live/index.js]: Lets comment this out but leave it in the file for testing. Essentially lets only merge working code

File: imports/pages/video/wowza.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+  static propTypes = {
+    dispatch: PropTypes.func.isRequired,
+    // params: PropTypes.object.isRequired,
+  }
+
+  componentWillMount() {
+    this.props.dispatch(navActions.setLevel("BASIC_CONTENT"));
+    this.props.dispatch(headerActions.set({
+      title: "Wowza",
```
Comment (jbaxleyiii) [inline imports/pages/video/wowza.js]: Lets make this "Live"

**Pull request merged**